kolibri/auth/models.py

```diff
@@ -489,8 +492,12 @@ def create_superuser(self, username, password):
         # get the default facility
         facility = Facility.get_default_facility()
 
+        if self.filter(username__iexact=username, facility=facility).exists():
+            raise ValidationError(_('An account with that username already exists'))
```

Comment:
Which username? :)

Reply:
That one! Where this is being used does present the username (this error is copied from the facility user serializer).